### PR TITLE
Updating ReadMe

### PR DESCRIPTION
**Pull request by pulkitaggarwl**
Updating the readme inline with other actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
+This repository contains GitHub Action for training machine learning models using Azure Machine Learning in a few different ways, each with different capabilities. To submit a training run, you have to define your python file(s) that should run remotely and file corresponding to [one of the supported methods of training](https://docs.microsoft.com/en-us/azure/machine-learning/concept-train-machine-learning-model#python-sdk)
 
-This action requires an AML workspace to be created or attached to via the [aml-workspace](https://github.com/Azure/aml-workspace) action and some compute resources to be available, which can be managed via the [aml-compute](https://github.com/Azure/aml-compute) action.
 
-## Template repositories
 
-This action is one in a series of actions that can be used to setup an ML Ops process. Examples of these can be found at
-1. Simple example: [ml-template-azure](https://github.com/machine-learning-apps/ml-template-azure) and
-2. Comprehensive example: [aml-template](https://github.com/Azure/aml-template).
+## Dependencies on other GitHub Actions
```
Comment (pulkitaggarwl) [inline README.md:17]: Leaving the heading as is for now. We can possibly have an offline conversation and update for all AzML actions together later

File: README.md [hunk 2]
```diff
@@ -66,7 +67,9 @@ jobs:
 | Input | Required | Default | Description |
 | ----- | -------- | ------- | ----------- |
 | azure_credentials | x | - | Output of `az ad sp create-for-rbac --name <your-sp-name> --role contributor --scopes /subscriptions/<your-subscriptionId>/resourceGroups/<your-rg> --sdk-auth`. This should be stored in your secrets |
-| parameters_file |  | `"compute.json"` | JSON file in the `.cloud/.azure` folder specifying your Azure Machine Learning compute target details. |
+| parameters_file |  | `"compute.json"` | We expect a JSON file in the .cloud/.azure folder in root of your repository specifying Azure Machine Learning compute target details. If you have want to provide these details in a file other than "compute.json" you need to provide this input in the action. |
+
+
 
 #### Azure Credentials
```
Comment (marvinbuss) [inline README.md:74]: Make the header consistent with other repos.

File: README.md [hunk 3]
```diff
@@ -107,15 +110,45 @@ A sample file can be found in this repository in the folder `.cloud/.azure`. The
...
+    - [Pipeline](https://docs.microsoft.com/en-us/python/api/azureml-pipeline-core/azureml.pipeline.core.pipeline%28class%29?view=azure-ml-py) or
+    - [AutoMLConfig](https://docs.microsoft.com/en-us/python/api/azureml-train-automl-client/azureml.train.automl.automlconfig.automlconfig?view=azure-ml-py)
+    
+    
+    
+| Parameter Name        | Required | Allowed Values           | Default    | Description |
+| --------------------- | -------- | ------------------------ | ---------- | ----------- |  
 | runconfig_python_file |          | str                      | `"code/train/run_config.py"`      | Path to the python script in your repository  in which you define your run and return an Estimator, Pipeline, AutoMLConfig or ScriptRunConfig object. |
 | runconfig_python_function_name |          | str                      | `"main"`              | The name of the function in your python script in your repository in which you define your run and return an Estimator, Pipeline, AutoMLConfig or ScriptRunConfig object. The function gets the workspace object passed as an argument. |
+
```
Comment (marvinbuss) [inline README.md:133]: Makes sense to move all this, but in the final document it looks very messy. @awmatheson thoughts?

**Comment by marvinbuss** [comment]
@pulkitaggarwl We merged the last feature request to master (download_artifacts) before next week. This also led to an update of the README file, which needs to be reflected here too.

File: README.md (hunk 3 above)
Comment (pulkitaggarwl) [reply README.md:133]: Made it further concise.

**Comment by pulkitaggarwl** [comment]
> @pulkitaggarwl We merged the last feature request to master (download_artifacts) before next week. This also led to an update of the README file, which needs to be reflected here too.

@marvinbuss  Is there some updates to ReadMe you see missing. I created a branch yesterday, so the latest changes should have been captured.

**Pull request merged**